keyboards/ploopyco/mouse/mouse.c

```diff
@@ -39,6 +39,18 @@
 #ifndef PLOOPY_DPI_DEFAULT
 #    define PLOOPY_DPI_DEFAULT 0
 #endif
+#ifndef PLOOPY_DRAGSCROLL_MOMENTARY
+#    define PLOOPY_DRAGSCROLL_MOMENTARY 0 // 0 -> Toggle, 1 -> Momentary
+#endif
+#ifndef PLOOPY_DRAGSCROLL_FIXED
+#    define PLOOPY_DRAGSCROLL_FIXED 0 // 0 -> Variable-DPI Drag Scroll, 1 -> Fixed-DPI Drag Scroll
+#endif
```

Comment:
Ideally, these shouldn't be defined at all.  Since they're a boolean, would be easier to check if they're defined or not.
```suggestion
```